docker-compose.yml

```diff
@@ -1,20 +1,23 @@
 version: '3.5'
 services:
   coordinator:
-   image: digitalcredentials/exchange-coordinator:0.1.0
-   # env_file:
-   #   - ./.coordinator.env
+    image: digitalcredentials/workflow-coordinator:0.1.0
+    env_file:
+      - ./.coordinator.env
     ports:
       - "4005:4005"
   signing:
```

Comment:
@jchartrand I have noticed that this is `signer` in some places (e.g., [here](https://github.com/digitalcredentials/docs/blob/main/deployment-guide/docker-compose-files/admin-dashboard-compose.yaml#L10)) and `signing` in others (e.g., [here](https://github.com/digitalcredentials/docs/blob/main/deployment-guide/docker-compose-files/simple-issuer-compose.yaml#L7)). I noticed a similar discrepancy with `transaction` and `transactions`. We should agree on a uniform naming convention. When it comes to `coordinator`, we should note that there is `issuer-coordinator` and `workflow-coordinator`. If any given deployment will only have one coordinator, keeping `coordinator` should be fine. However, if these two could co-exist or if we anticipate adding other coordinators, we should reconsider this name as well.

Reply:
This is good to align on in docs, but does not affect Docker Hub deployments, which only care about the `image` property specified in each service.